cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,16 @@
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
+
+    cy.visit('https://conduit.mate.academy/user/register');
+
+    cy.get(':nth-child(1) > .form-control').type(user.username);
+    cy.get(':nth-child(2) > .form-control').type(user.email);
+    cy.get(':nth-child(3) > .form-control').type(user.password);
```

Comment:
Instead of using ':nth-child()' selector which is not very readable and maintainable, consider using more specific selectors. For example, if the input fields have 'name' or 'id' attributes, you could use those for selecting. This way, even if the structure of your HTML changes, your tests will still work.